apps/frontend/src/app/pages/jobs-details/jobs-details.component.html

```diff
@@ -0,0 +1,52 @@
+<div class="container mx-auto min-h-screen px-3 sm:px-0">
+  <div class="md:flex gap-8">
+    <div>
+      <div
+        class="flex gap-2 items-center py-4 cursor-pointer"
+        [routerLink]="['/pages/jobs']"
+      >
+        <i class="bi bi-arrow-left text-base-100 text-xl"></i>
+        <p class="font-bold text-sm text-base-100 font-mono">Back</p>
+      </div>
+      <div class="pt-6 flex gap-6 pb-8">
+        <div class="flex gap-2">
+          <i class="bi bi-clock text-primary"></i>
+          <p class="font-mono text-base-200">{{ time }}</p>
+        </div>
+        <div class="flex gap-2">
+          <i class="bi bi-laptop text-primary"></i>
+          <p class="font-mono text-base-200">{{ locale }}</p>
+        </div>
+      </div>
+      <div class="pb-8">
+        <h3 class="font-sans font-medium text-2xl text-base-100 pb-2">
+          {{ name }}
+        </h3>
+        <p class="font-mono text-base-200">{{ responsibilities }}</p>
+      </div>
+      <div class="pb-10">
+        <p class="font-mono font-bold text-base-100 pb-2">Description</p>
+        <p class="font-mono text-base-200">{{ description }}</p>
+      </div>
+      <button class="btn btn-primary w-[200px]">Apply</button>
+    </div>
+    <div class="pt-20">
+      <div
+        class="bg-neutral md:w-[416px] md:h-[336px] w-full h-full rounded-lg p-8"
+      >
+        <div class="flex items-center pb-6">
+          <img
```

Comment:
The image and the company name are too close:

![Captura de tela de 2024-03-13 15-13-37](https://github.com/Venture-Miner/wagademy/assets/83927877/742efc63-7d48-4060-adba-efcf346c4f6d)